ui/src/main/webapp/package.json

```diff
@@ -33,25 +33,30 @@
     "@angular/platform-browser": "^2.4.5",
     "@angular/platform-browser-dynamic": "^2.4.5",
     "@angular/router": "^3.4.5",
-    "bootstrap": "3.3.6",
-    "bootstrap-datepicker": "1.4.0",
-    "bootstrap-select": "1.7.3",
-    "bootstrap-switch": "3.3.2",
+    "angular2-moment": "^1.1.0",
```

Comment:
I suggest to keep the entries in an alphabetical order in the first place. I only used npm.